miso-web/src/spring-test/java/uk/ac/bbsrc/tgac/miso/webapp/springtest/BarcodableSearchRestControllerST.java

```diff
@@ -0,0 +1,14 @@
+package uk.ac.bbsrc.tgac.miso.webapp.springtest;
+
+import org.junit.Test;
+import java.util.Arrays;
+
+public class BarcodableSearchRestControllerST extends AbstractST {
+
+  private static final String CONTROLLER_BASE = "/rest/barcodables";
+
+  @Test
+  public void testSearch() throws Exception {
+    baseSearchByTerm(CONTROLLER_BASE + "/search", "11111", Arrays.asList(1));
```

Comment:
This returns `BarcodableDtos` which can be created from several different entity types, so checking the ID is not enough to verify the results. You should check both the ID and the `entityType` field.